src/libraries/System.IO.Compression/src/System/IO/Compression/CompressionLevel.cs

```diff
@@ -3,27 +3,19 @@
 
 namespace System.IO.Compression
 {
-    /// <summary>
-    /// Specifies values that indicate whether a compression operation emphasizes speed or compression size.
-    /// </summary>
-
-    // This is an abstract concept and NOT the ZLib compression level.
-    // There may or may not be any correspondence with the a possible implementation-specific level-parameter of the deflater.
+    /// <summary>Specifies values that indicate whether a compression operation emphasizes speed or compression size.</summary>
+    /// <remarks><format type="text/markdown"><![CDATA[
+    /// [!INCLUDE[remarks](~/includes/remarks/System.IO.Compression/CompressionLevel/CompressionLevel.md)]
+    /// ]]></format></remarks>
     public enum CompressionLevel
     {
-        /// <summary>
-        /// The compression operation should balance compression speed and output size.
-        /// </summary>
+        /// <summary>The compression operation should be optimally compressed, even if the operation takes a longer time to complete.</summary>
```

Comment:
Old and new sentences are describing completely different behaviors, do you know if the new one is actually true?

Reply:
Yeah, it seems it should be written as:

```csharp
/// <summary>The compression operation should be optimally compressed, balancing compression speed and output size.</summary>
```